### DISCUSSION TITLE
Disable responsive modifies on custom plugin

### DISCUSSION THREAD

**Discussion by AndreaChiabotto** [tailwindlabs/tailwindcss#14261] [Help]
Hi,
i'm looking for a way to write a plugin and disable the responsive modifiers, so that the class .foo can be used, but md:foo or lg:foo are not generated. I know that there was a way in tailwind 2, is there a workaroung in tailwind 3?

 plugin(function ({ addComponents }) {
      addUtilities(
        {
          '.foo': {
            margin: '0 auto',
            'max-width': '1152px',
            'background-color': '#ffcc00' }
      );
    }),

**Comment by wongjn** [comment]
Yes, add the classes you don't want to be generated in the [`blocklist`](https://tailwindcss.com/docs/content-configuration#discarding-classes):

```js
/** @type {import('tailwindcss').Config} */
module.exports = {
  blocklist: [
    'md:foo',
    'lg:foo',
  ],
  // …
}
```
Though `md:foo` and `lg:foo` won't be generated anyway, unless those classes are found as strings in files that Tailwind scans.
